source/extensions/filters/http/ext_proc/processor_state.h

```diff
@@ -141,6 +170,9 @@ class ProcessorState : public Logger::Loggable<Logger::Id::filter> {
   bool trailers_available_ : 1;
   // If true, then a CONTINUE_AND_REPLACE status was used on a response
   bool body_replaced_ : 1;
+  // If true, we are in "buffered partial" mode and already got back
+  // the response to the buffer.
+  bool partial_limit_reached_ : 1;
```

Comment:
The variable seems to speak of hitting a limit but the comments refers to a specific state, is there something missing here or is it talking about the same thing?

Reply:
I agree that name is confusing. I named it "partial_body_processed" and I think that is a bit clearer.